chore: Add LayerFeaturesCount interface oc:5097

peco11 opened the pull request:
This commit adds the LayerFeaturesCount interface to the codebase. It includes properties for tracking the number of tracks and points of interest (pois) in each layer.

peco11 committed:
chore: Add LayerFeaturesCount interface

This commit adds the LayerFeaturesCount interface to the codebase. It includes properties for tracking the number of tracks and points of interest (pois) in each layer.

peppedeka commented on src/feature.ts:
```diff
@@ -74,3 +74,10 @@ export interface WmProperties {
   success: 'media deleted';
 }
+
+export interface LayerFeaturesCount {
```
creare l'interfaccia per {track , poi} e usare record come per l'environment

peco11 committed:
chore: Add type definitions for LayerFeaturesCount and LayerFeatureCount

- Added type definition for LayerFeaturesCount interface
- Added type definition for LayerFeatureCount interface
- Updated the structure of LayerFeaturesCount to use Record instead of index signature
